src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -23,11 +24,11 @@ object WdlExpression {
       case t: Terminal if t.getTerminalStr == "identifier" =>
         lookup(t.getSourceString)
       case t: Terminal if t.getTerminalStr == "integer" =>
-        new WdlValue(t.getSourceString.toInt, WdlIntegerType)
+        new WdlIntegerValue(t.getSourceString.toInt)
```

Comment:
bump these all back up to the case line. IMO putting things on the next line are for when you have multiline cases